docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -134,20 +127,13 @@ Each target can have one or more [compilations](#compilations).
-| `artifactsTaskName` | The name of the task that builds the resulting artifacts of this target.                                                                                                                                                                                                                        |
-| `components`        | The components used to setup Gradle publications.                                                                                                                                                                                                                                               |
-| `compilerOptions`   | The [compiler options](gradle-compiler-options.md) used for the target. This declaration overrides any `compilerOptions {}` configured at [top level](multiplatform-dsl-reference.md#top-level-blocks). To use it, add the following opt-in: `@OptIn(ExperimentalKotlinGradlePluginApi::class)` |
-
-> The support for `compilerOptions {}` as a common target configuration is [Experimental](components-stability.md#stability-levels-explained)
-> and requires opt-in. It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate
-> your feedback on it in [YouTrack](https://kotl.in/issue).
->
-{style="warning"}
+| **Name**            | **Description**                                                                                                                                                                                | 
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `attributes`        | Attributes used for [disambiguating targets](multiplatform-set-up-targets.md#distinguish-several-targets-for-one-platform) for a single platform.                                              |
+| `platformType`      | Designates the Kotlin platform of this target. Available values: `jvm`, `androidJvm`, `js`, `wasm`, `native`, `common`.                                                                        |
+| `artifactsTaskName` | The name of the task that builds the resulting artifacts of this target.                                                                                                                       |
+| `components`        | The components used to setup Gradle publications.                                                                                                                                              |
+| `compilerOptions`   | The [compiler options](#compiler-options) used for the target. This declaration overrides any `compilerOptions {}` configured at [top level](multiplatform-dsl-reference.md#top-level-blocks). |
```

Comment:
```suggestion
| `compilerOptions`   | [Compiler options](#compiler-options) used for the target. This declaration overrides any `compilerOptions {}` configured at the [top level](multiplatform-dsl-reference.md#top-level-blocks). |
```

Reply:
I accepted half of this one. I didn't include "_the_ top level". It sounds more natural without "the" as there is only one top level.